lms/djangoapps/courseware/module_render.py

```diff
@@ -758,9 +757,7 @@ def rebind_noauth_module_to_user(module, real_user):
     # As we have the time to manually test more modules, we can add to the list
     # of modules that get the per-course anonymized id.
     is_pure_xblock = isinstance(descriptor, XBlock) and not isinstance(descriptor, XModuleDescriptor)
-    module_class = getattr(descriptor, 'module_class', None)
-    is_lti_module = not is_pure_xblock and issubclass(module_class, LTIModule)
```

Comment:
This is not needed any longer since `LTIBlock` `is_pure_xblock`.

Reply:
Hurray for fewer special cases!